libraries/mapping/engines/cesium/src/lib/utils/cesiumSetup.ts

```diff
@@ -16,3 +17,21 @@ export const setupCesiumEnvironment = (config?: CesiumConfig) => {
   const baseUrl = config?.baseUrl ? config.baseUrl : getDefaultBaseUrl();
   window.CESIUM_BASE_URL = baseUrl;
 };
+
+export const getIsViewerReadyAsync = async (
```

Comment:
currently only used in cesium-reference playground, should be used to rerender responsive UI, when the viewer is ready to get proper state.